apps/frontend/src/pages/onboarding/libs/components/onboarding-answer/styles.module.css

```diff
@@ -1,9 +1,4 @@
 .onboarding-answer {
-	width: 342px;
-}
-
-@media (width <= 640px) {
-	.onboarding-answer {
-		width: 250px;
-	}
+	width: 100%;
+	max-width: 342px;
```

Comment:
According to figma design width of answers for desktop screen is 342px but for mobile screen it is 312px.